### PR TITLE
refactor: add financial mutation attributes to statements

### PR DESCRIPTION
**Pull request by TimEvci**

### CODE REVIEW

**Comment by ghost** [comment]
There seems to be a test that is failing

**Comment by TimEvci** [comment]
> There seems to be a test that is failing

Fixed it, the mock import was incorrect, so it tried to connect to the Moneybird Client.

File: moneysnake/financial_statement.py [hunk 1]
```diff
@@ -18,6 +19,32 @@ class FinancialStatement(MoneybirdModel):
...
+            financial_statement_data.pop("financial_mutations", [])
+        )
+
+        if self.id is None:
+            data = post_request(
+                f"{self.endpoint}s",
+                data={self.endpoint: financial_statement_data},
+                method="post",
+            )
+            self.update(data)
```
Comment (ghost) [inline moneysnake/financial_statement.py]: You can move the self.update to after the if-else statement, you might need to define the data variable before the if-else statement with proper typing (If strict is applied)

**Comment by ABDreos** [comment]
Only one minor comment!

**Review by ghost — CHANGES_REQUESTED** [review]
Minor fix!

File: moneysnake/financial_statement.py (hunk 1 above)
Comment (TimEvci) [reply moneysnake/financial_statement.py]: Updated!

**Review by ghost — APPROVED** [review]

**Pull request merged**